Modules/Microsoft365DSC/DSCResources/MSFT_TeamsUpdateManagementPolicy/MSFT_TeamsUpdateManagementPolicy.schema.mof

```diff
@@ -0,0 +1,14 @@
+[ClassVersion("1.0.0.0"), FriendlyName("TeamsUpdateManagementPolicy")]
+class MSFT_TeamsUpdateManagementPolicy : OMI_BaseResource
+{
+    [Key, Description("Identity of the Teams Update Management Policy.")] String Identity;
+    [Write, Description("The description of the Teams Update Management Policy.")] String Description;
+    [Write, Description("Determines if managed updates should be allowed or not.")] Boolean AllowManagedUpdates;
+    [Write, Description("Determines if preview builds should be allowed or not.")] Boolean AllowPreview;
+    [Write, Description("Determines the ring of public previews to subscribes to."), ValueMap{"Disabled","Enabled","FollowOfficePreview"}, Values{"Disabled","Enabled","FollowOfficePreview"}] String AllowPublicPreview;
+    [Write, Description("Determines the day of week to perform the updates. Value shoud be between 0 and 6.")] UInt32 UpdateDayOfWeek;
```

Comment:
If a value between 0 and 6 is expected, shouldn't this be checked in the code? Or an ValidateRange be added?